general/1_terminal/Leonid-Kharin/task4.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash jq
+
+file=$(cat result.json)
+
+echo "please, wait 15 seconds"
+
+# iterate over all keys with value text
+
+for ((i=0; i >(-1); i++)) do 
```

Comment:
I don't think it's a good idea to wait for the number to overflow before stopping.

This may work correctly, it's not very clean. The `for` loop is definitely not the most elegant solution.